lib/browse-terms/index.js

```diff
@@ -108,6 +108,9 @@ const determineUpdatedTerms = async (esResourcesProperty, idsToFetch, freshBibDa
   const liveTerms = await fetchPropertyForUris(idsToFetch, esResourcesProperty)
   const terms = freshBibData.map((esBibDoc, idx) => {
     const freshTerms = getSubjectModels(esBibDoc)
+    if (process.env.FORCE_ALL_SUBJECTS === 'true') {
```

Comment:
as noted offline, this is doing the same thing as the INGEST_BROWSE_MODE env var [here](https://github.com/NYPL/research-catalog-indexer/blob/d924b13ed888bffc5048fc1d489be7b2ccde6945/lib/browse-terms/index.js#L93). we don't need to hit determineUpdatedTerms at all if we are only sending the subjects from the bib data